hook.go

```diff
@@ -19,28 +19,92 @@ package hook
 #cgo linux LDFLAGS: -lX11-xcb -lxcb -lxcb-xkb -lxkbcommon -lxkbcommon-x11
 //#cgo windows LDFLAGS: -lgdi32 -luser32
 
-#include "event/goEvent.h"
 // #include "event/hook_async.h"
+#include "event/goEvent.h"
+
 */
 import "C"
 
 import (
-	// 	"fmt"
-	"unsafe"
+	"time"
 )
 
-// AddEvent add event listener
-func AddEvent(key string) int {
```

Comment:
Please keep the compatibility of the original code.

Reply:
Do you mean that it should check if the key is pressed at that exact moment? the previous behavior was blocking until the key is pressed, but i'm not so sure that is desirable.

What about adding a callback to the AddEvent function?

Reply:
I mean don't break the compatibility of the api, if it's new features keep the original code.